### ISSUE TITLE
Pod::Html

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#642]
Migrated from [rt.perl.org#1515](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=1515) (status was 'resolved')

Searchable as RT1515$

**Comment by p5pRT** [comment]
### From The RT System itself

The problem is with "pod2html" formatting of "=item text" entries\.  From
what I see\, this text&#8203;:



---
=over 4

=item hi there

The definition of "hi there" is xyzzy\.

=back


---
will be formatted as \(minus a few extra CRs\)&#8203;:

\<DL>
\<DT>\<STRONG>\<A NAME="item\_hi">hi there\</A>\</STRONG>
\<DD>\<P>The definition of "hi there" is xyzzy\.
\</DL>

This should\, from my reading of HTML\, instead be formatted as&#8203;:

\<DL>
\<DT>\<STRONG>\<A NAME="item\_hi">hi there\</A>\</STRONG>\</DT>
\<DD>The definition of "hi there" is xyzzy\.\</DD>
\</DL>

In other words\, there is an extra paragraph break and the DT and DD are not
closed properly\.  This does make a difference in the output\!

\-\-
David Masterson
\*    KLA\-Tencor Corp\.
\*    Yield Management Systems
\*    408\-875\-6836
\*    David\.Masterson@&#8203;kla\-tencor\.com

**Comment by p5pRT** [comment]
### From @schwern

Appears fixed in 5\.8\.1 RC2's pod2html\, probably fixed eariler\.

**Comment by p5pRT** [comment]
@schwern - Status changed from 'stalled' to 'resolved'
